x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/trusted_apps_page.tsx

```diff
@@ -157,7 +177,7 @@ export const TrustedAppsPage = memo(() => {
     >
       <TrustedAppsNotifications />
 
-      {isCheckingIfEntriesExists ? (
+      {isCheckingIfEntriesExists && !didEntriesExist ? (
```

Comment:
Needed to avoid blank page when there is no results until we know if the search result is empty or not exists any trusted app.

Reply:
Just to make sure:

1. no entries exists at all, regardles if there was a filter or not, we show the empty state
2. entries exists, but current filter did not return any items - we show "no items found" type of message

I'm not really clear on what `didEntriesExist` is. Can you confirm the UI behavirour

Reply:
This is to avoid blank page (and hiding the policy selector) when checking if entries exists.
So first time you go to that page it will hide the searchException bar and will show the loader (as expected) since there is no entries already (prev loaded).
After that, if you set some filter that returns no results, when it will check if exists or not entries, the searchException bar will still present until we know if exists or not any entry not returned by the search performed before.

What didEntriesExist means is if there was some entry before the current search (prevState).